src/app/services/evt-status.service.ts

```diff
@@ -179,6 +182,15 @@ export class EVTStatusService {
             params,
         };
     }
+
+    getPageElementsByClassList(classList) {
+        const pageContent = this.currentStatus$.pipe(
+            map(({ page }) => page.parsedContent));
+
+        return pageContent.pipe(map((x) => deepSearch(x, 'class', classList)));
```

Comment:
You can have directly a return without the `pageContent` variable. Just pipe `map`s you need to `this.currentStatus$`. 
Moreover, use a more self explaining name instead of `x`